src/main/java/bhs/devilbotz/commands/DriveCommand.java

```diff
@@ -8,16 +9,18 @@
 public class DriveCommand extends CommandBase {
   private final DriveTrain drive;
   private final DoubleSupplier speed;
-  private final DoubleSupplier rotation;
```

Comment:
Why not leave it as "rotation"?  It's easier to read/understand, but no big deal.

Reply:
I can put it back, but more WPILIB methods use rot instead of rotation